tox.ini

```diff
@@ -269,7 +269,7 @@ deps =
     consul10: python-consul>=1.0,<1.1
     consul11: python-consul>=1.1,<1.2
     ddtracerun: redis
-    test_django: pytest-django
+    test_django: pytest-django==3.10.0
```

Comment:
```suggestion
    test_django: pytest-django~=3.10
```
?

or

```suggestion
    test_django: pytest-django>=3.10.0,<4.0.0
```

?

Reply:
I think it'd be better to pin it to a specific version since it's a util.